src/panels/lovelace/entity-rows/hui-error-entity-row.ts

```diff
@@ -0,0 +1,45 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+class HuiErrorEntityRow extends LitElement {
+  public entity?: string;
+
+  static get properties() {
+    return {
+      entity: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entity) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <div>
+        Entity not available: ${this.entity}
+      </div>
+    `;
+  }
+
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        div {
```

Comment:
Oh, one more comment. Skip the whole div. Remove it from the template and update CSS to style  `:host`

```css
:host {
  display: block;
  background-color: yellow;
  padding: 8px;
}
```

btw do we know why we use `flex: 1`?

Reply:
I do not

Reply:
I can see that we only work with the best engineers here 👍 

Switch to `:host` was good, I removed the `<div>` as it is no longer needed.

Reply:
Hey, you brought on a C++ dev that start front-end stuff three months ago 🤣
I am very much the "well it works" crowd right now, but am trying to learn the why as well.